### DISCUSSION TITLE
How to disable all the on-paste stuff?

### DISCUSSION THREAD

**Discussion by aapeliv** [redhat-developer/vscode-java#3157] [Q&A]
Having a lot of issues with paste actions not working. It's extremely frustrating when you can't copy-paste normally.

What are the config options to disable those features?

[provide a description of the issue]

##### Environment
- Operating System:
- JDK version:
- Visual Studio Code version:
- Java extension version:

##### Steps To Reproduce
1. [step 1]
2. [step 2]

[Please attach a sample project reproducing the error]
[Please attach logs](https://github.com/redhat-developer/vscode-java/wiki/Troubleshooting#enable-logging)

**Comment by sebastien-allard** [comment]
Let's remember this one:

[Please do not override paste](https://github.com/redhat-developer/vscode-java/issues/1115)

**Comment by sebastien-allard** [comment]
It is actually a new VSCode feature that reactivated the [Please do not override paste](https://github.com/redhat-developer/vscode-java/issues/1115) issue:
https://code.visualstudio.com/updates/v1_79#_paste-as

To paste the normal way:
Shift + Ctrl + V

To disable it:
"editor.pasteAs.enabled": "false"
"editor.pasteAs.showPasteSelector": "never"

**Reply by ViniciusFXavier** [reply]
Thks for the help

**Comment by tiagofrancafernandes** [comment]
On settings set `editor.pasteAs.enabled` as `false`


Help set too:

```json
"typescript.updateImportsOnPaste.enabled": false,
"javascript.updateImportsOnPaste.enabled": false,
```


Final set:

```json
"editor.formatOnPaste": false,
"editor.pasteAs.enabled": false,
"typescript.updateImportsOnPaste.enabled": false,
"javascript.updateImportsOnPaste.enabled": false,
"editor.pasteAs.showPasteSelector": "never",
```
